byzcoin/viewchange_test.go

```diff
@@ -276,6 +277,7 @@ func TestViewChange_LostSync(t *testing.T) {
 //  - Node0 - leader - stopped after creation of block #1
 //  - Node3 - misses block #1, unpaused after creation of block #1
 func TestViewChange_NeedCatchUp(t *testing.T) {
+	t.Skip("Flaky test - skipping it")
```

Comment:
Good question - keep it or remove it? Supposing that in April it goes into "low maintenance mode", I kind of would like to keep the tests around, just in case.

On the other hand, they are stored somewhere in the git-history.

I have a small preference to keeping them. If you have a strong preference to remove the flaky tests, I'll delete them.

Reply:
Nah, keep it, low maintenance as you said.